content/posts/2014/11/17/services-announcement.md

```diff
@@ -1,10 +1,10 @@
---- 
+---
```

Comment:
We've been using Anope since 2014 and have lots of relevant information elsewhere.